pkg/standalone/runfileconfig/run_file_config.go

```diff
@@ -13,23 +13,101 @@ limitations under the License.
 
 package runfileconfig
 
-import "github.com/dapr/cli/pkg/standalone"
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/dapr/cli/pkg/standalone"
+)
+
+const (
+	appLogFileNamePrefix   = "app"
+	daprdLogFileNamePrefix = "daprd"
+	logFileExtension       = ".log"
+)
 
 // RunFileConfig represents the complete configuration options for the run file.
 // It is meant to be used with - "dapr run --run-file <path-to-run-file>" command.
 type RunFileConfig struct {
 	Common  Common `yaml:"common"`
-	Apps    []Apps `yaml:"apps"`
+	Apps    []App  `yaml:"apps"`
 	Version int    `yaml:"version"`
 }
 
-// Apps represents the configuration options for the apps in the run file.
-type Apps struct {
+// App represents the configuration options for the apps in the run file.
+type App struct {
 	standalone.RunConfig `yaml:",inline"`
 	AppDirPath           string `yaml:"app_dir_path"`
+	appLogFile           *os.File
+	daprdLogFile         *os.File
 }
 
 // Common represents the configuration options for the common section in the run file.
 type Common struct {
 	standalone.SharedRunConfig `yaml:",inline"`
 }
+
+func (a *App) GetLogsDir() string {
+	logsPath := filepath.Join(a.AppDirPath, ".dapr", "logs")
```

Comment:
Nit: `".dapr"` and `"logs"` should be constants as well?

Reply:
done